synfig-studio/src/gui/instance.cpp

```diff
@@ -102,6 +104,8 @@ Instance::Instance(synfig::Canvas::Handle canvas, synfig::FileSystem::Handle con
 {
 	id_=instance_count_++;
 
+	set_clear_redo_stack_on_new_action(true);
```

Comment:
Is it required here? You will set it at `Instance::create` anyway.

Reply:
I never know if somebody will write a `new Instance()` in internal code XD